### DISCUSSION TITLE
New 9.1 weeklys

### DISCUSSION THREAD

**Discussion by bchevreau** [SavedInstances/SavedInstances#521] [Ideas]
Hi,
Absolute lover of this addon, I couldn't play without it.

I was wondering if we could expect the addition of Covenant maw assaults and weekly Torments of Torghast (it is daily but the soul cinders is locked on a weekly basis same as Wrath of the Jailer) coming to the addon in the near future?

I would suggest replacing Nzoth assaults and/or horrific visions by these 2, as they are quite important and, to my knowledge, not easily trackable for altholics like myself.

Much love and long live SavedInstances!

**Comment by LiangYuxuan** [comment]
Will be in next alpha.

**Reply by zalidthebur** [reply]
In the meantime I did stumble on how to add Tormentors (Thanks go to Fzzl on wowhead) :

If you're using SavedInstances, a quick hack is to add this line to the function local _SpecialQuests {}, starts on line 15, in the file Quests.lua (located in _retail_\Interface\AddOns\SavedInstances\Modules).
[63854] = { name=L["Tormentors Of Torghast"] }, -- Tormentors Of Torghast"

**Reply by LiangYuxuan** [reply]
It's now on 9.1.1. No longer require editing yourself. :)

**Reply by zalidthebur** [reply]
Nice one. I totally never thought to check before using that hack :-D
